### PR TITLE
Delete associated trigger at the end of the run

### PR DESCRIPTION
**Pull request by y-lohse**
Aside from the account, we also want to delete the trigger for the connector.

### CODE REVIEW

File: src/CozyUtils.js [hunk 1]
```diff
@@ -144,6 +145,26 @@ class CozyUtils {
       return accountsCollection.destroy(doc)
     }
   }
+
+  async deleteTrigger(accountId) {
+    const triggersCollection = this.client.collection(DOCTYPE_TRIGGERS)
+    const response = await triggersCollection.find({
+      worker: 'konnector'
```
Comment (y-lohse) [inline src/CozyUtils.js]: There is no route to do a multi field query on the triggers, we can only filter by `worker`.

File: src/CozyUtils.js [hunk 2]
```diff
@@ -144,6 +145,26 @@ class CozyUtils {
...
+    })
+    const triggers = response.data
+    const accountTriggers = triggers.filter(trigger => {
+      return (
+        get(trigger, 'message.konnector') === 'toutatice' &&
+        get(trigger, 'message.account') === accountId
+      )
+    })
+
+    await Promise.all(
```
Comment (y-lohse) [inline src/CozyUtils.js]: No `plimit` here — in theory there's only one trigger matching the criteria.

**Comment by y-lohse** [comment]
[(𝐒) [Connecteur Toutatice] 🎓- OIDC - logout](https://trello.com/c/zXear0YU/1959-%F0%9D%90%92-connecteur-toutatice-%F0%9F%8E%93-oidc-logout)

File: src/index.js [hunk 3]
```diff
@@ -106,6 +106,7 @@ async function start(fields) {
     })
 
     log('info', 'Deleting account')
+    await cozyUtils.deleteTrigger(accountId)
     await cozyUtils.deleteAccount(accountId)
```
Comment (Crash--) [inline src/index.js]: we can run the deletion in parallel ? await Promise.all([])

File: src/CozyUtils.js (hunk 1 above)
Comment (Crash--) [reply src/CozyUtils.js]: :-°

File: src/CozyUtils.js [hunk 4]
```diff
@@ -144,6 +145,27 @@ class CozyUtils {
...
+      // Int theory there is only one matching trigger, so there shoudn't be many promises here
```
Comment (Crash--) [inline src/CozyUtils.js]:
```suggestion
      // In theory there is only one matching trigger, so there shouldn't be many promises here
```

**Pull request merged**